x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -345,103 +352,55 @@ export class ReportingStore {
     return body;
   }
 
-  public async setReportCompleted(
+  public async setReportError(
     report: SavedReport,
-    completedInfo: ReportCompletedFields
+    failedInfo: ReportFailedFields
   ): Promise<UpdateResponse<ReportDocument>> {
-    const { output } = completedInfo;
-    const status =
-      output && output.warnings && output.warnings.length > 0
-        ? JOB_STATUS.WARNINGS
-        : JOB_STATUS.COMPLETED;
     const doc = sourceDoc({
-      ...completedInfo,
-      status,
-    } as ReportSource);
+      ...failedInfo,
+      status: JOB_STATUS.PROCESSING,
```

Comment:
I'm testing this now, but I _think_ we don't want to update the status field in this update:

```suggestion
      ...errorInfo,
```